dv/verilator/memutil/cpp/verilator_memutil.h

```diff
@@ -21,6 +21,7 @@ enum MemImageType {
 struct MemArea {
   std::string name;      // Unique identifier
   std::string location;  // Design scope location
+  int width;             // Memory width
```

Comment:
How about making that size_t as well and remove the cast in `VerilatorMemUtil::MemWrite()`?